.gitignore

```diff
@@ -138,3 +138,6 @@ dmypy.json
 
 # Open API Schema file
 /open_api.yaml
+
+# Nano temporary files
+*.swp
```

Comment:
I think you can use global gitignore file for ignoring files that are dependent solely on your system/editor
https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/

Reply:
I like that.  I'll try it out and revert